dotnet/src/SemanticKernel.Test/AI/OpenAI/Services/AzureOpenAIConfigTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
These files and other in this test project are bad merge results, I believe you just need to delete this and BackendConfigTests, OpenAIConfigTests, and CodeBlockTests.

Reply:
Okay will do.